### PR TITLE
[OND211-2039] implement automatic release

### PR DESCRIPTION
**Pull request by MaxWolf-01**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,55 @@
-# Fully automated build and deploy process for ondewo-csi-client-python
+# Fully automated build and deploy process for ondewo-t2s-client-python
```
Comment (foldvaridominic) [inline Makefile]: accidental typo

**Review by foldvaridominic — APPROVED** [review]

File: Dockerfile.utils [hunk 2]
```diff
@@ -0,0 +1,61 @@
+FROM python:3.8-slim
```
Comment (xavierad) [inline Dockerfile.utils:1]: I am not sure if we should use the ones from the registry... @teddius ?

File: RELEASE.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
 # Release History
 *****************
 
+## Release ONDEWO CSI Python Client 2.9.0
+
+### New Features
+* [[OND211-2039]](https://ondewo.atlassian.net/browse/OND211-2039) - Improved automated release process
+* Upgraded to CSI API Version 2.0.0
```
Comment (xavierad) [inline RELEASE.md:8]: Does not have ticket? Same comment from above

File: RELEASE.md [hunk 4]
```diff
@@ -1,6 +1,14 @@
 # Release History
 *****************
 
+## Release ONDEWO CSI Python Client 2.9.0
+
+### New Features
+* [[OND211-2039]](https://ondewo.atlassian.net/browse/OND211-2039) - Improved automated release process
```
Comment (xavierad) [inline RELEASE.md]: I don't know what is usual in CSI, but what is the subject? As far as I know, the release notes should contain notes in the following structure: "ticket" - "subject". "description"

**Pull request merged**